gui/src/services/api.ts

```diff
@@ -26,13 +26,13 @@ axios.interceptors.request.use(
   config => {
     const constStr = require("../common/const-string.json");
 
-    if (config.headers.auth_token) {
+    if (config.headers?.auth_token) {
       config.headers.Authorization = config.headers.auth_token;
       delete config.headers.auth_token;
     } else {
       const token = localStorage.getItem(constStr.access_token);
       if (token) {
-        config.headers.Authorization = token;
+        config.headers!.Authorization = token;
```

Comment:
Should this not be ?

Reply:
You can assign the value to this property only if it is present. Since optional chaining operator can result in undefined, it cannot be used.